uncrustify.cfg

```diff
@@ -60,19 +60,25 @@ sp_cmt_cpp_start			= add
 sp_cmt_cpp_doxygen			= true
 sp_angle_shift				= remove
 sp_permit_cpp11_shift		= true
+sp_before_sparen			= true
+sp_after_operator			= remove
+sp_after_operator_sym		= remove
 
 sp_before_ptr_star			= add
 sp_before_byref				= add
 sp_after_ptr_star			= remove
 sp_after_byref				= remove
-sp_before_unnamed_ptr_star	= remove
-sp_before_unnamed_byref		= remove
-sp_after_ptr_star_func		= add
-sp_after_byref_func			= add
-sp_before_ptr_star_func		= remove
-sp_before_byref_func		= remove
+sp_before_unnamed_ptr_star	= add
+sp_before_unnamed_byref		= add
+sp_after_ptr_star_func		= remove
```

Comment:
In `TorrentInfo &operator=(const TorrentInfo &other)`  it aligns `&` to the function name.